modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/CoreStatistics_conf.pm

```diff
@@ -130,38 +130,65 @@ sub pipeline_analyses {
                          },
       -max_retry_count => 1,
       -flow_into       => {
-                            '2' => ['GeneRelatedTasks'],
+                            '2' => ['CheckAssemblyGenesetGeneTasks'],
                           },
       -rc_name         => 'normal',
     },
-
+    { -logic_name  => 'CheckAssemblyGenesetGeneTasks',
+      -module      => 'Bio::EnsEMBL::Production::Pipeline::Common::CheckAssemblyGeneset',
+      -parameters  => {
+          run_all => $self->o('run_all'),
+          release => $self->o('release')
+       },
+      -can_be_empty    => 1,
+      -flow_into       => ['GeneRelatedTasks'],
+      -max_retry_count => 1,
+      -hive_capacity   => 10,
+      -priority        => 5,
+      -rc_name         => 'normal',
+    },
     {
       -logic_name      => 'GeneRelatedTasks',
       -module          => 'Bio::EnsEMBL::Hive::RunnableDB::Dummy',
       -max_retry_count => 1,
       -flow_into       => {
-                            '1->A' => [
+                            '1->A' => WHEN( '#new_assembly# >= 1 or #new_genebuild# >=1' => [
```

Comment:
Why not put this one level further up, in the CheckAssemblyGenesetGeneTask's flow to 'GeneRelatedTasks'? That would simplify the logic a bit, only need one conditional.
Same argument applies to CheckAssemblyGenesetChromosomeTasks.

Reply:
Very good point, these dummy are not really needed anymore. I've cleaned up these analysis.